web/blueprints/user/tables.py

```diff
@@ -46,92 +39,76 @@ class MembershipTable(BootstrapTable):
     In the toolbar, a “new membership” button is inserted if the
     :py:obj:`current_user` has the ``add_membership`` property.
     """
+    group_name = Column("Gruppe")
+    begins_at = Column("Beginn", formatter='table.dateFormatter')
+    ends_at = Column("Ende", formatter='table.dateFormatter')
+    actions = Column("Aktionen", formatter='table.multiBtnFormatter')
```

Comment:
A `MultiBtnColumn` and `DateColumn` could be worth to think about.

Reply:
Resolved in 5689730222cbb88cada268b21b2d65ce5730afc9, d75faddd19c5ae967bf2344117d55ee7d0b34655